### ISSUE TITLE
RPN for COCO OVD setting

### ISSUE THREAD

**Issue by yhcao6** [microsoft/RegionCLIP#41]
Dear author,

Thanks for your great work. May I ask if the RPN used to pre-train the RegionCLIP is trained on the LVIS dataset for both COCO and LVIS OVD settings? If the answer is yes, is it fair for the COCO OVD setting? Because there are some overlaps between the base classes of LVIS and the novel classes of COCO, and you use some box annotations of novel classes to train the RPN.

**Comment by YiwuZhong** [comment]
@yhcao6 Please refer to Implementation Details in paper: "The RPN used in transfer learning was trained by the base categories of target dataset (e.g., the transfer learning on COCO used the RPN trained on COCO)".

**Comment by yhcao6** [comment]
@YiwuZhong I mean in the pretraining stage that trains the RegionCLIP. You use LVIS-trained RPN to obtain proposals on caption datasets right?

**Comment by YiwuZhong** [comment]
@yhcao6 Yes, during pretraining, the proposals come from an RPN which was trained by the boxes of LVIS base categories, without using their categorical labels.
